lib/mongo/crypt/handle.rb

```diff
@@ -52,6 +52,31 @@ def ref
 
       private
 
+      # Send the logs from libmongocrypt to the Mongo::Logger
+      def set_logger
+        @log_callback = Proc.new do |level, msg|
+          logger = Mongo::Logger.logger
+
+          case level
+          when :fatal
+            logger.fatal(msg)
+          when :error
+            logger.error(msg)
+          when :warning
+            logger.warn(msg)
+          when :info
+            logger.info(msg)
+          when :trace
+            logger.debug(msg)
```

Comment:
Would it make sense to have the level symbols match the Ruby logger methods, to simplify the mapping?

Reply:
That's awesome, thank you.